### ISSUE TITLE
Context Menus: Focus border invisible on first focus 

### ISSUE THREAD

**Issue by BrittanyIRL** [swissspidy/web-stories-wp#11311]
This one is a little weird, we have 2 context menu usages in the Canvas that aren't showing the focus of the first focusable element even though they are focused. The right click menu and the floating menu, when you navigate into them with the keyboard, should show the first element that's enabled with the focus outline. You can hit 'enter' and the element that should be focused will be changed so it's definitely the active element, the focus style just isn't getting applied. 

https://user-images.githubusercontent.com/1820266/164103850-f7ecce9a-b494-4155-b17c-129284eb6837.gif

## Acceptance Criteria 
- When keyboard is used to enter a context men (right click, floating menu) the focus outline should be visible on the first element.
